### PR TITLE
Bugfix/#5533-added-services

### PR DESCRIPTION
**Pull request by KKatiaa**
#5533 Change request to US #4427, #4428 "Add a service "Сервіс" to the pricing card", "Edit a service "Сервіс" in the pricing card"
https://github.com/ita-social-projects/GreenCity/issues/5533

### CODE REVIEW

File: src/app/ubs/ubs/services/order.service.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { Order } from '../models/ubs.model';
 import { UBSOrderFormService } from './ubs-order-form.service';
 import { OrderClientDto } from 'src/app/ubs/ubs-user/ubs-user-orders-list/models/OrderClientDto';
 import { ResponceOrderFondyModel } from '../../ubs-user/ubs-user-orders-list/models/ResponceOrderFondyModel';
+import { serializeNodes } from '@angular/compiler/src/i18n/digest';
```
Comment (katia9261) [inline src/app/ubs/ubs/services/order.service.ts]: Where do we use this import?

File: src/app/ubs/ubs/services/order.service.ts [hunk 2]
```diff
@@ -157,4 +158,12 @@ export class OrderService {
   saveOrderData(): void {
     this.localStorageService.setOrderWithoutPayment(true);
   }
+
+  saveServiceName(serviceName: string): void {
```
Comment (katia9261) [inline src/app/ubs/ubs/services/order.service.ts]: Where do we use these functions and is there any reason to redeclare them?

**Review by katia9261 — CHANGES_REQUESTED** [review]

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.ts [hunk 3]
```diff
@@ -128,6 +128,10 @@ export class UbsAdminTariffsAddServicePopUpComponent implements OnInit, OnDestro
     this.loadingAnim = false;
   }
 
+  getControl(control: string) {
+    return this.addServiceForm.get(control);
```
Comment (atsa21) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.ts:132]: Add type of function return :AbstractControl

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.html [hunk 4]
```diff
@@ -50,8 +50,13 @@ <h4>{{ 'ubs-tariffs-add-service.edit_service' | translate }}</h4>
           formControlName="description"
           id="description"
           placeholder="{{ 'ubs-tariffs-add-service.placeholder-text' | translate }}"
+          maxlength="255"
+          required
         >
         </textarea>
+        <div class="validation" *ngIf="getControl('description').invalid && getControl('description').touched">
+          {{ 'ubs-tariffs-add-service.error_service_field_message' | translate }}
```
Comment (atsa21) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.html:58]: Use **app-ubs-input-error** for ubs error messages
For example we use it in **UBSAddAddressPopUpComponent**

**Pull request closed without merging**